### PR TITLE
Add initial run for local files

### PR DESCRIPTION
**Pull request by liamawhite**
The UX is pretty awful/has no value add right now. This is part of an iterative process towards a bare-bones `getenvoy list/pull/run`.

Signed-off-by: Liam White <liam@tetrate.io>

### CODE REVIEW

File: pkg/binary/run.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	"fmt"
+	"os"
+	"syscall"
+)
+
+func Run(path string, args []string) error {
+	if _, err := os.Stat(path); err != nil {
+		return fmt.Errorf("unable to stat %q: %v", path, err)
+	}
+	if err := syscall.Exec(path, append([]string{"envoy"}, args...), os.Environ()); err != nil {
```
Comment (taiki45) [inline pkg/binary/run.go]: Q: Can we make sure the `path` will always end with `envoy` filename? If not, we should use a filename from the `path` for the first element of the args for execve(2).

Comment (liamawhite) [reply pkg/binary/run.go]: We should not assume this, good catch.

File: pkg/cmd/root.go [hunk 2]
```diff
@@ -21,9 +21,11 @@ import (
...
 func NewRoot() *cobra.Command {
-	return &cobra.Command{
-		Use:   "getenvoy",
-		Short: "getenvoy",
-		Long:  "getenvoy",
-	}
+	return rootCmd
+}
+
+var rootCmd = &cobra.Command{
```
Comment (taiki45) [inline pkg/cmd/root.go]: nit: Are these changes intentional or mis-changes from rebasing? I think this should be separately committed for the record and for the better reviews (OK for this time).

Comment (liamawhite) [reply pkg/cmd/root.go]: These are intentional changes due to a rebase.
